### ISSUE TITLE
The context in beforeScenario/afterScenario doesn't contain the scenario name

### ISSUE THREAD

**Issue by Shazwazza** [artilleryio/artillery#932]
It would be helpful if the context or event data for beforeScenario/afterScenario contained the scenario name. I am using these methods to initialize and clear custom code for streaming event counters and need to record for which scenario it is.

**Comment by hassy** [comment]
good shout @Shazwazza, those hooks should probably have access to the entire scenario object

**Comment by bernardobridge** [comment]
This feature is now released in [Artillery v2.0.0-38](https://github.com/artilleryio/artillery/releases/tag/v2.0.0-38).

You have access to the whole scenario object through `context.scenario`. For example:

```
function myBeforeScenarioHandler(context, ee, next) {
    //access context.scenario to obtain any property of the scenario script
    ee.emit('counter', `my_custom_counter_${context.scenario.name}`);
    next();
};
```
